menas/ui/service/RestDAO.js

```diff
@@ -17,13 +17,29 @@ class RestClient {
 
   static cache = _.memoize($.ajax, ({url}) => url);
 
-  static get(url, shouldUseCache = false) {
+  static _setBusy(oCtl) {
```

Comment:
I'd like to maintain separation of concerns and keep `RestClient` and `RestDAO` unaware of any `ui5` controls.